Dockerfile

```diff
@@ -37,7 +37,7 @@ ENV CHROME_BIN /usr/local/bin/chromium-no-sandbox
 ENV CHROMIUM_BIN /usr/local/bin/chromium-no-sandbox
 
 ### downgrade node version due to nodegit error with v0.7.0
-RUN sudo npm install -g n && sudo n 6.9.0
+RUN sudo npm install -g n yarn && sudo n 6.9.0
```

Comment:
nitpick: this is a weird place to put this as the comment is:

> ### downgrade node version due to nodegit error with v0.7.0

Considering that the version of node that is action at `npm install` time can change what actually gets installed, you may want to move this to a separate layer.